libs/ui/src/hooks/use-activity.hook.ts

```diff
@@ -31,50 +32,109 @@ const transformApiData = (
   chainName: string
 ): SectionListActivityData[] => {
   let response: SectionListActivityData[] = [];
-  const filteredTransactions = data?.history_list.filter(
-    txData => !(txData.receives.length > 0 && txData.sends.length > 0) && txData.cate_id !== 'approve'
-  );
 
   let sectionListItem: SectionListActivityData | undefined;
 
   const userTokensMetadata: ActivityResponse['token_dict'] = data.token_dict;
+  const userProjectsMetadata: ActivityResponse['project_dict'] = data.project_dict;
```

Comment:
why do you need explicit types for `userProjectsMetadata` and `userTokensMetadata`?  
types should be auto resolved from `data` type